maven-plugin/pom.xml

```diff
@@ -6,30 +6,35 @@
     <version>1.27-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
-  <artifactId>maven-localizer-plugin</artifactId>
+  <artifactId>localizer-maven-plugin</artifactId>
```

Comment:
make this change now - as it will break in the future as the Apache Maven trademark forbids maven-**-plugin.